test/e2e/specs/editor/various/style-variation.spec.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'adding blocks', () => {
+	test( 'Should switch to the plain style of the quote block', async ( {
+		page,
+		pageUtils,
+	} ) => {
+		await pageUtils.createNewPost();
+
+		// Inserting a quote block
+		await pageUtils.insertBlock( { name: 'core/quote' } );
+		await page.keyboard.type( 'Quote content' );
+
+		await pageUtils.clickBlockToolbarButton( 'Quote' );
+
+		const plainStyleButton = await page.locator(
+			'button[role="menuitem"]:has-text("Plain")'
```

Comment:
This should be using role selectors. (I know currently it lacks documentation on how to write role selectors. Hopefully, Playwright will soon publish their official doc.)

```suggestion
			'role=menuitem[name="Plain"i]'
```

Reply:
Thanks